### PR TITLE
Fix bug when manually adding users crashed

### LINKED ISSUE #1242: User creation in the admin is broken
Sentry Issue: [CONCREXIT-3F](https://sentry.io/organizations/thalia/issues/1844597243/?referrer=github_integration)

```
FieldError: Unknown field(s) (password2, password1) specified for User
  File "django/contrib/admin/options.py", line 702, in get_form
    return modelform_factory(self.model, **defaults)
  File "django/forms/models.py", line 554, in modelform_factory
    return type(form)(class_name, (form,), form_class_attrs)
  File "django/forms/models.py", line 267, in __new__
    raise FieldError(message)

FieldError: Unknown field(s) (password2, password1) specified for User. Check fields/fieldsets/exclude attributes of class UserAdmin.
(15 additional frame(s) were not displayed)
...
  File "django/utils/decorators.py", line 130, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "django/contrib/admin/options.py", line 1522, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "django/contrib/admin/options.py", line 1555, in _changeform_view
    ModelForm = self.get_form(request, obj, change=not add)
  File "django/contrib/auth/admin.py", line 80, in get_form
    return super().get_form(request, obj, **defaults)
  File "django/contrib/admin/options.py", line 704, in get_form
    raise FieldError(
```

### PR DESCRIPTION
**Pull request by PeaceDucko**
Closes #1242

### Summary
When wanting to manually add a user in the /admin dashboard, the site would throw an error where it didn't recognise the password1 and password2 fields. This PR solves above issue.

### How to test
Steps to test the changes you made:
1. Log in as an administrative user
2. Go to the admin dashboard
2. Under 'Authentication and Authorization' click on 'Add' next to the 'Users' tab
3. Site shows the form to create a new user and does not crash

### CODE REVIEW

File: website/members/forms.py [hunk 1]
```diff
@@ -54,44 +54,19 @@ class UserCreationForm(BaseUserCreationForm):
 
     # Don't forget to edit the formset in admin.py!
     # This is a stupid quirk of the user admin.
-
-    # shadow the password fields to prevent validation errors,
-    #   since we generate the passwords dynamically.
-    password1 = None
-    password2 = None
+    password1 = forms.CharField(label='Password', widget=forms.PasswordInput)
+    password2 = forms.CharField(label='Repeat Password', widget=forms.PasswordInput)
```
Comment (se-bastiaan) [inline website/members/forms.py:58]: I'm not sure but I think this can all go away :)

File: website/members/forms.py [hunk 2]
```diff
@@ -54,44 +54,19 @@ class UserCreationForm(BaseUserCreationForm):
...
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
```
Comment (se-bastiaan) [inline website/members/forms.py:61]: If the init is not overridden anymore it can be removed

File: website/members/forms.py [hunk 3]
```diff
@@ -54,44 +54,19 @@ class UserCreationForm(BaseUserCreationForm):
...
-        password = get_user_model().objects.make_random_password(length=15)
-        # pass the password on as if it was filled in, so that save() works
-        self.cleaned_data["password1"] = password
-        user = super().save(commit=False)
-        user.set_password(password)
-        if commit:
-            user.save()
-        if self.cleaned_data["send_welcome_email"]:
-            language = settings.LANGUAGE_CODE
-            emails.send_welcome_message(user, password, language)
```
Comment (se-bastiaan) [inline website/members/forms.py:90]: This email is no longer used so it can be removed as well!

**Review by se-bastiaan — CHANGES_REQUESTED** [review]
Overall you have the perfect idea of what should happen, just not done removing all that is possible! :D

File: website/members/forms.py [hunk 4]
```diff
@@ -38,28 +38,7 @@ class Meta:
         ]
         model = Profile
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
```
Comment (se-bastiaan) [inline website/members/forms.py:42]: Woah, this init must not go, the widget is set to hidden here and we do not want to lose that functionality.

**Review by se-bastiaan — CHANGES_REQUESTED** [review]

**Comment by se-bastiaan** [comment]
It seems you also require a rebase!

File: website/members/forms.py (hunk 3 above)
Comment (se-bastiaan) [reply website/members/forms.py:90]: This is located in the members/emails.py file.

File: website/members/forms.py (hunk 4 above)
Comment (PeaceDucko) [reply website/members/forms.py:42]: That's a big 'whoops'

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**